composer/utils/inference.py

```diff
@@ -101,6 +101,9 @@ def export_for_inference(
     # make a copy of the model so that we don't modify the original model
```

Comment:
Let us raise an error in this function if the model is ddp wrapped (same as we do for deepspeed wrapped).